### ISSUE TITLE
Product block: Images should display at the same ratio 

### ISSUE THREAD

**Issue by vivialice** [woocommerce/woocommerce-gutenberg-products-block#8338]
When images are added to products, they don't show the same ratio in the Products block with landscape images. Ideally, there would be controls on the Product Image block to customize the ratio and size, but by default, all should display at the same size. 

Example screenshot:
<img width="633" alt="Screenshot 2023-01-30 at 19 11 12" src="https://user-images.githubusercontent.com/9888201/215572540-d29cad3c-d55b-4002-bc4d-b10fd7fac55d.png">

**Comment by Aljullu** [comment]
For reference, Gutenberg has a similar issue for the Image block: https://github.com/WordPress/gutenberg/issues/38990.

**Comment by richtabor** [comment]
Also, aspect ratio has been added to the Post Featured Image block (https://github.com/WordPress/gutenberg/pull/47854) with design follow-ups here (https://github.com/WordPress/gutenberg/issues/47963). 🚀

**Comment by danielwrobert** [comment]
FYI: this may also be impacted and/or blocked by woocommerce/woocommerce#42478.

**Comment by kmanijak** [comment]
Closing this one as a duplicate of https://github.com/woocommerce/woocommerce/issues/42415 which has a bit more context.
